script/filter-package-json.py

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env python3
+
+import json
+
+# We need a list of packages that are devDependencies but not needed to build
+with open('package-exclude.json') as package_exclude:
+    excludes = json.load(package_exclude)
+
+def is_excluded(package):
+    return (package in excludes['EXCLUDE_NPM_PACKAGES']
+            or any(package.startswith(prefix) for prefix in excludes['EXCLUDE_NPM_PREFIXES']))
+
+def filter_section(packages, section):
+    deps = packages.get(section, {})
+    for package, version in sorted(deps.items()):
+        if not is_excluded(package):
+            yield (package, version)
```

Comment:
Just a thought:
```suggestion
def filter_section(packages, section):
    deps = packages.get(section, {})
    return {package: version for package, version in sorted(deps.items()) if not is_excluded(package)}
```

You could also make deps a direct parameter and do `data[section]` a level higher. So:
```suggestion
def filter_dependencies(deps):
    return {package: version for package, version in sorted(deps.items()) if not is_excluded(package)}
```

Reply:
I like the later!